src/main/java/nl/dtls/fairdatapoint/vocabulary/DCAT3.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * The MIT License
+ * Copyright © 2017 DTL
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package nl.dtls.fairdatapoint.vocabulary;
+
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.ValueFactory;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+
+public class DCAT3 {
+
+    public static final ValueFactory VF = SimpleValueFactory.getInstance();
+    public static final String PREFIX = "dcat";
+    public static final String NAMESPACE = "http://www.w3.org/ns/dcat#";
```

Comment:
We could borrow the actual values from the rdf4j vocab maybe? Or we inherit from that file (I don't know by heart how they structured those files, maybe it's a `final` class)

Reply:
Improved in [fixup! Change dcterms:hasVersion to dcat:version](https://github.com/FAIRDataTeam/FAIRDataPoint/pull/442/commits/c7c55cea4910edf7bb948050eb272c5c694de413) (will be squashed before merge)